src/styles/bs4/summernote-bs4.js

```diff
@@ -1,121 +1,173 @@
 import $ from 'jquery';
-import '/js/settings.js';
-import renderer from '/js/renderer';
+
+import '../../js/settings.js';
```

Comment:
Vite also supports the base folder alias (like `@`) and we're already using it in testing codes against Vitest. Could you let us know what will be used between the relative or the absolute paths?

Reply:
While introducing vitest to `/test/**`, we are referencing `/src/**` path with `@/`. However, because `/src/**` is still being built with Webpack during bundling, it cannot be referenced with `@` for now. It would be good to apply the same rules after migrating Webpack to Vite.

Reply:
We can also alias in webpack, but we don't recommend it for this PR. 

```js
  resolve: {
    roots: [path.resolve('./src')],
    alias: {
      '@': path.resolve('./src'),
    },
  },
```

Reply:
Got it. Then I'll apply similar alias rules for Vite later. Thanks.